app/assets/javascripts/jquery.slick.js

```diff
@@ -308,9 +358,9 @@
         });
 
       } else {
-
         _.applyTransition();
-        targetLeft = Math.ceil(targetLeft);
+
+        targetLeft = Math.floor(targetLeft);
```

Comment:
A couple of notes regarding this call: 

- The need to swap from `ceil` to `floor` was discovered empirically (using `floor` does a better job getting rid of the edges on all the scenarios I tested).  

- I'm sure this is not the case but given that `TargetLeft` is always 0 or a negative value I wonder if the author of this function overlooked that when dealing with negative values `ceil` and `floor` roles are exchanged (if we forget to pay attention to the sign).

![Screen Shot 2021-01-04 at 19 59 27](https://user-images.githubusercontent.com/1012038/103588175-62eba180-4ec7-11eb-86d2-307632166096.png)